src/background.js

```diff
@@ -9,6 +9,15 @@ const BRANCHES = {
   TREATMENT: "treatment",
 };
 
+// The example.com domains are for testing. Consumers of isDefaultEngineHomepage
+// should check that the active hostname is not example.com before displaying
+// a search tip.
```

Comment:
Adding these example.com domains is a little awkward, but I didn't see another way to test this functionality given
1. Mochitests won't let load actual google.com etc. URLs
2. We (presumably) want to test the actual search engines we'll be using and not a fake search engine created for the purposes of the test.

Reply:
There's a `MOZ_DISABLE_NONLOCAL_CONNECTIONS` environment variable that defaults to `1`. But when I set it to `0`, the test runner crashes with an exception. Would be nice to figure this out because I imagine an end-to-end test for this experiment would want to load all the real homepages. I looked around on Bugzilla but nobody else seems to be having this problem.

    Traceback (most recent call last):
      File "/Users/adw/mc/obj-artifact/_tests/testing/mochitest/runtests.py", line 2874, in doTests
        e10s=options.e10s
      File "/Users/adw/mc/obj-artifact/_tests/testing/mochitest/runtests.py", line 2300, in runApp
        self.execute_start_script()
      File "/Users/adw/mc/obj-artifact/_tests/testing/mochitest/runtests.py", line 1772, in execute_start_script
        return self.marionette.execute_script(script, script_args=(self.start_script_kwargs, ))
      File "/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
        self.gen.throw(type, value, traceback)
      File "/Users/adw/mc/testing/marionette/client/marionette_driver/marionette.py", line 1332, in using_context
        self.set_context(scope)
      File "/Users/adw/mc/testing/marionette/client/marionette_driver/marionette.py", line 1309, in set_context
        {"value": context})
      File "/Users/adw/mc/testing/marionette/client/marionette_driver/decorators.py", line 26, in _
        return func(*args, **kwargs)
      File "/Users/adw/mc/testing/marionette/client/marionette_driver/marionette.py", line 587, in _send_message
        raise errors.InvalidSessionIdException("Please start a session")
    InvalidSessionIdException: Please start a session
     0:03.64 ERROR Automation Error: Received unexpected exception while running application

Reply:
I got the same error with `MOZ_DISABLE_NONLOCAL_CONNECTIONS` :/  There's this bug about DISABLE_NONLOCAL and Marionette https://bugzilla.mozilla.org/show_bug.cgi?id=1371576 -- maybe the issue is broader than Marionette and DISABLE_NONLOCAL just always crashes?